docs/core/install/linux-ubuntu.md

```diff
@@ -0,0 +1,336 @@
+
+```bash
+wget https://packages.microsoft.com/config/ubuntu/17.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
+sudo dpkg -i packages-microsoft-prod.deb
+```
+
+[!INCLUDE [linux-apt-install-21](includes/linux-apt-install-21.md)]
+
+## 16.10 ❌
+
+[!INCLUDE [linux-not-supported](includes/linux-ubuntu-not-supported.md)]
+
+[!INCLUDE [linux-apt-prep-intro](includes/linux-ubuntu-apt-prep-intro.md)]
+
+```bash
+wget https://packages.microsoft.com/config/ubuntu/16.10/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
+sudo dpkg -i packages-microsoft-prod.deb
+```
+
+[!INCLUDE [linux-apt-install-21](includes/linux-apt-install-21.md)]
+
+## APT update SDK or runtime
+
+When a new patch release is available for .NET Core, you can simply upgrade it through APT with the following commands:
+
+```bash
+sudo apt-get update
+sudo apt-get upgrade
+```
+
+## APT troubleshooting
+
+This section provides information on common errors you may get while using APT to install .NET Core.
+
+### Unable to locate
+
+If you receive an error message similar to **Unable to locate package {netcore-package}**, run the following commands.
+
+There are two placeholders in the following set of commands.
+
+- `{netcore-package}`\
+This represents the .NET Core package you're installing, such as `20.04`. This is in the `sudo apt-get install` command.
+
+- `{ubuntu-version}`\
+This represents the Ubuntu version you are on. This is in the `wget` command.
+
+Try purging the package list:
+
+```bash
+sudo dpkg --purge packages-microsoft-prod && sudo dpkg -i packages-microsoft-prod.deb
+sudo apt-get update
+sudo apt-get install {netcore-package}
+```
+
+If that doesn't work, you can run a manual install with the following commands:
+
+```bash
+sudo apt-get install -y gpg
+wget -O - https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor -o microsoft.asc.gpg
+sudo mv microsoft.asc.gpg /etc/apt/trusted.gpg.d/
+wget https://packages.microsoft.com/config/ubuntu/{ubuntu-version}/prod.list
+sudo mv prod.list /etc/apt/sources.list.d/microsoft-prod.list
+sudo chown root:root /etc/apt/trusted.gpg.d/microsoft.asc.gpg
+sudo chown root:root /etc/apt/sources.list.d/microsoft-prod.list
+sudo apt-get install -y apt-transport-https
+sudo apt-get update
+sudo apt-get install {netcore-package}
+```
+
+### Failed to fetch
+
+[!INCLUDE [package-manager-failed-to-fetch-deb](includes/package-manager-failed-to-fetch-deb.md)]
+
+## Snap
+
+[.NET Core is available from the Snap Store.](https://snapcraft.io/dotnet-sdk)
+
+A snap is a bundle of an app and its dependencies that works without modification across many different Linux distributions. Snaps are discoverable and installable from the Snap Store. For more information about Snap, see [Getting started with Snap](https://snapcraft.io/docs/getting-started).
+
+Only supported versions of .NET Core are available through Snap. Use the following commands to install.
```

Comment:
I think the rest of the doc has commands just after "following commands" so I expect that, but here it's after a few more paragraphs--which I guess is valid, but not sure if it's intentional.